src/applications/personalization/profile-2/routes.js

```diff
@@ -4,7 +4,7 @@ import MilitaryInformation from './components/MilitaryInformation';
 import DirectDeposit from './components/direct-deposit/DirectDeposit';
 import ConnectedApplications from './components/connected-apps/ConnectedApps';
 
-const getRoutes = showDirectDeposit => {
+const getRoutes = (showDirectDeposit, showMilitaryInformation) => {
```

Comment:
It won't matter in this case since we are only calling this function once and you updated that call, but in most situations I'd worry about adding a second arg and not defaulting it to `true` since it would introduce potentially unexpected behavior if this function was called a bunch of places with only one arg.